pom.xml

```diff
@@ -1399,6 +1400,11 @@ under the License.
         <artifactId>gson</artifactId>
         <version>2.8.9</version>
       </dependency>
+      <dependency>
```

Comment:
Adding OWASP is out of scope, please revert

Reply:
ok, removed. I will override this dependency in another way.